.github/workflows/beekeeper.yml

```diff
@@ -14,7 +14,7 @@ env:
   SETUP_CONTRACT_IMAGE: "ethersphere/bee-localchain"
   SETUP_CONTRACT_IMAGE_TAG: "0.9.2"
   BEELOCAL_BRANCH: "main"
-  BEEKEEPER_BRANCH: "master"
+  BEEKEEPER_BRANCH: "feat/feed-wrapping"
```

Comment:
Beekeeper [PR](https://github.com/ethersphere/beekeeper/pull/469) should be merged together with these changes. Also, `BEEKEEPER_BRANCH` should be `master` when merged.